onegov/election_day/formats/common.py

```diff
@@ -142,3 +142,44 @@ def load_csv(
         )
 
     return csv, error
+
+
+# Verification utils
+def line_is_relevant(line, number, district=None):
+    if district:
+        return line.sortwahlkreis == district and line.sortgeschaeft == number
+    else:
```

Comment:
You can remove the else, since the if above always returns:

```python
if district:
    return line.sortwahlkreis == district and line.sortgeschaeft == number

return line.sortgeschaeft == number
```